### PR TITLE
dev-ml/camlp5: version bump to 7.10

### PR DESCRIPTION
**Pull request by donahue95**
signed-off-by: Drake Allen Donahue donahue95@comcast.net

### CODE REVIEW

**Comment by donahue95** [comment]
econf failed, presume upstream package is slightly non standard

File: dev-ml/camlp5/camlp5-7.10.ebuild [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+DEPEND="
+	<=dev-lang/ocaml-4.6.0:=[ocamlopt?]
+"
+RDEPEND="${DEPEND}"
+
+S="${WORKDIR}/${PN}-rel$(ver_rs 1- '')"
+
+src_prepare() {
+	eapply_user
+}
```
Comment (juippis) [inline dev-ml/camlp5/camlp5-7.10.ebuild]: This should be default behaviour and not needed here.

File: dev-ml/camlp5/camlp5-7.10.ebuild [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+EAPI="7"
+
+MY_P=${P%_p*}
+DESCRIPTION="A preprocessor-pretty-printer of ocaml"
+HOMEPAGE="https://camlp5.github.io/"
+SRC_URI="https://github.com/camlp5/camlp5/archive/rel$(ver_rs 1- '').tar.gz -> ${P}.tar.gz"
+LICENSE="BSD"
+SLOT="0/${PV}"
+KEYWORDS="~alpha ~amd64 ~ppc ~x86"
+IUSE="doc +ocamlopt"
```
Comment (juippis) [inline dev-ml/camlp5/camlp5-7.10.ebuild]: What does `doc` IUSE do in this ebuild?

File: dev-ml/camlp5/camlp5-7.10.ebuild [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+"
+RDEPEND="${DEPEND}"
+
+S="${WORKDIR}/${PN}-rel$(ver_rs 1- '')"
+
+src_prepare() {
+	eapply_user
+}
+src_configure() {
+	./configure \
```
Comment (juippis) [inline dev-ml/camlp5/camlp5-7.10.ebuild]: Please add a comment above this saying this is not autotools-compatible configure script.

File: dev-ml/camlp5/camlp5-7.10.ebuild [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+}
+src_compile(){
+	emake out
+	if use ocamlopt; then
+		emake opt
+		emake opt.opt
+	fi
+}
+src_install() {
+	if [[ -f Makefile ]] || [[ -f GNUmakefile ]] || [[ -f makefile ]] ; then
```
Comment (juippis) [inline dev-ml/camlp5/camlp5-7.10.ebuild]: This check seems rather useless. What will you do if none of these files exist? Why wouldn't they exist? I think you can just remove the whole `src_install` phase and it should install files just fine.

File: dev-ml/camlp5/camlp5-7.10.ebuild (hunk 1 above)
Comment (donahue95) [reply dev-ml/camlp5/camlp5-7.10.ebuild]: Yes.

File: dev-ml/camlp5/camlp5-7.10.ebuild (hunk 2 above)
Comment (donahue95) [reply dev-ml/camlp5/camlp5-7.10.ebuild]: Nothing, gone.

File: dev-ml/camlp5/camlp5-7.10.ebuild (hunk 3 above)
Comment (donahue95) [reply dev-ml/camlp5/camlp5-7.10.ebuild]: Ys.

File: dev-ml/camlp5/camlp5-7.10.ebuild (hunk 4 above)
Comment (donahue95) [reply dev-ml/camlp5/camlp5-7.10.ebuild]: It does.

**Comment by atupone** [comment]
7.10 is now bumped

**Pull request closed without merging**